src/config/config.rs

```diff
@@ -362,6 +365,23 @@ fn read_extra_versions(toml: &toml::Value) -> Result<Vec<Version>, String> {
     }
 }
 
+fn read_extra_dox_features(toml: &toml::Value) -> Result<Vec<String>, String> {
+    match toml.lookup("options.extra_dox_features") {
```

Comment:
`toml.lookup(...).and_then(|a| your code).unwrap_or_else(Vec::new)` is probably nicer and shorter

Reply:
Hum, no. The code is quite horrible. ^^'